depth_grid:
  odom_topic: "/lidar_slam/odom"
  cloud_topic: "/pcl_render_node/cloud"
#  cloud_topic: "/global_cloud"
  bisec_num: 10
  resolution: 0.1
  sensing_range: 10
  point_filt_num: 2
#  angular_reso: 0.1
  vertical_fov: 100
  horizon_fov: 100
#  vertical_fov: 70.4
#  horizon_fov: 77.4